### PR TITLE
#43-Feature/post linking the frontend to backend- Saadia & Chioma [MAIN]

### LINKED ISSUE #43: Linking the frontend to backend (POST)
## Story

- **As a** developer

- **I Want** to link the post endpoints to the frontend

- **So That** I can add new data to the database from the client side.

## Acceptance Criteria

1. Given a student redirected to the dashboard student 
   - When he add a feedback
   - Then I can see a new feedback added to the student_feedback table
2. Given a mentor redirected to the dashboard mentor
   - When he add a feedback
   - Then I can see a new feedback added to the mentor_feedback table

## Implementation Notes

1. Create endpoint on the server side
2. Test it on postman 
3. Link it to client side

## Checklist:

- [ ] I have defined the user story
- [ ] I have defined some acceptance criteria
- [ ] I have included any implementation notes
- [ ] I have included all relevant links

### PR DESCRIPTION
**Pull request by SaadiaELF**
## Naming Rules

Name your PR like this: ISSUENUMBER-TITLE-YOURNAME

## Description

Write a summary of your change. Explain why you have made it.

## Related to

Make sure you include the issue number with a hash sign # so Github can link this PR to the right issue, like this:

Fixes #43

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have carefully reviewed my own code
- [ ] I have commented my code
- [ ] I have updated any documentation

### CODE REVIEW

**Review by Arbeits-Sachen — APPROVED** [review]

File: server/api.js [hunk 1]
```diff
@@ -97,4 +98,55 @@ router.get("/modules", async (req, res) => {
...
+		let modules = await db.query(
+			"SELECT * FROM users u INNER JOIN student_feedback sf ON u.user_id = sf.student_id INNER JOIN modules m ON sf.module_id = m.module_id WHERE u.user_id = $1 ORDER BY sf.date DESC",
+			[userId]
+		);
+		return res.json(modules.rows[0]);
+	} catch (error) {
+		console.log(error);
+	}
+});
+router.post("/feedback/student", auth, async (req, res) => {
```
Comment (JanefrancessC) [inline server/api.js:113]: Is there a reason you added auth to this? because it disrupts the system. Can you re-check to see if the system does not break?

Comment (SaadiaELF) [reply server/api.js:113]: I forgot to add the header on the front end, I'll do it

Comment (SaadiaELF) [reply server/api.js:113]: Fixed

**Review by JanefrancessC — APPROVED** [review]
Great job!

**Pull request merged**